features/openhab-addons/pom.xml

```diff
@@ -10,7 +10,7 @@
     <version>3.0.0-SNAPSHOT</version>
   </parent>
 
-  <artifactId>org.openhab.addons.features.karaf.openhab-addons</artifactId>
+  <artifactId>org.openhab.addons.features.karaf.openhab-addons3</artifactId>
```

Comment:
why change this?

Reply:
To avoid the name clash between the auto-ported 2.5.x feature and the new one in 3.0.